src/runtime.js

```diff
@@ -190,6 +175,19 @@ function adoptShared(global, callback) {
    */
   global.AMP.setTickFunction = (unusedFn, opt_flush) => {};
 
+  /**
+   * Installs services for the given AMP document (used for
+   * shadow documents and amp-next-page)
+   * @export
+   */
+  global.AMP.installAmpdocServices = installAmpdocServices.bind(null);
+
+  /**
+   * Exports the combined (shared and doc) styles
+   * @export
+   */
+  global.AMP.combinedCss = ampDocCss + ampSharedCss;
```

Comment:
/cc @lannka

Reply:
We don't need `amp-next-page` inside `amp4ads` so I'll just exclude the `a4a` binary from needing to export this and that should bring back its size to what it used to be

Reply:
Done